.github/workflows/build-kata-static-tarball-riscv64.yaml

```diff
@@ -0,0 +1,118 @@
+name: CI | Build kata-static tarball for riscv64
+on:
+  workflow_call:
+    inputs:
+      stage:
+        required: false
+        type: string
+        default: test
+      tarball-suffix:
+        required: false
+        type: string
+      push-to-registry:
+        required: false
+        type: string
+        default: no
+      commit-hash:
+        required: false
+        type: string
+      target-branch:
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  build-asset:
+    runs-on: riscv-builder
```

Comment:
We need `riscv-builder` added to https://github.com/kata-containers/kata-containers/blob/main/.github/actionlint.yaml as a "known runner"